containers/daap-create-metadata/src/var/task/create_metadata.py

```diff
@@ -0,0 +1,64 @@
+import json
+import os
+
+from data_platform_logging import DataPlatformLogger
+from data_product_metadata import DataProductMetadata
+
+
+def handler(event, context):
+
+    logger = DataPlatformLogger(
+        extra={
+            "image_version": os.getenv("VERSION", "unknown"),
+            "base_image_version": os.getenv("BASE_VERSION", "unknown"),
+            "lambda_name": context.function_name,
+        }
+    )
+
+    try:
+        data_product_name = event["metadata"]["name"]
+    except KeyError:
+        logger.error("The name of the data product is missing")
+        return {
```

Comment:
Not sure yet, but maybe we could have some helper functions to format these kinds of error responses? I think we might end up reusing some of these messages.

Also, from the perspective of someone using the API, they might find it useful to have a machine readable error code that is more specific than the HTTP status code (but we could worry about this later)